ner/src/main/java/edu/illinois/cs/cogcomp/ner/ExpressiveFeatures/GazetteerTree.java

```diff
@@ -35,7 +36,16 @@
     private StringSplitterInterface splitter = new StringSplitterInterface() {
         @Override
         public String[] split(String line) {
-            return line.split("[\\s]+");
+
+            // character tokenization for Chinese
+            if(ParametersForLbjCode.currentParameters.language.equals("zh")) {
```

Comment:
better would be https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/tree/master/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/constants